torch/_export/logical_schema.py

```diff
@@ -267,14 +272,11 @@ class Graph:
     nodes: List[Node]
 
     # Tensor values that appear in the graph
-    # They could be graph inputs, graph outputs, or intermediate values produced by nodes
-    # The key is a unique identifider name for the IValue
-    # The name will be used in the graph and node to refer to the IValue
-    ivalues: Dict[str, IValue]
+    # They could be graph inputs, graph outputs, or intermediate tensor values produced by nodes
+    tensor_values: List[TensorValue]
 
-    # SymInts that appear in the graph
-    # Key is the name/identifier of the SymInt, not the expression of the SymInt
-    symint_values: Dict[str, SymInt]
+    # SymInt values that appear in the graph
+    symint_values: List[SymIntValue]
```

Comment:
An alternative design is to merge `tensor_values` and 'symint_values' into a single list. 
However, I am leaning towards keeping them separated.

Reply:
We plan to have SymInt, SymFloat, and SymBool. Do you think they should all be separate fields?

Reply:
No. I want one field for tensor_values, one field for sym_values. The rationale being, if downstream system doesn't care about dynamic shape, or doesn't intend to implement sym* stuff, everything in the sym_values field are the "problems" they need to address. 

e.g.
They can specialize sym_value with some concrete shape. 
They can convert sym_ints into "Tensor Ints".